frontend/packages/ux-editor-v3/src/components/config/editModal/EditGrid/EditGrid.test.tsx

```diff
@@ -4,7 +4,7 @@ import userEvent from '@testing-library/user-event';
 import { EditGrid } from './EditGrid';
 import { renderWithMockStore, renderHookWithMockStore } from '../../../../testing/mocks';
```

Comment:
Alias can be added to line 6 and 8 😊 
```suggestion
import { renderWithMockStore, renderHookWithMockStore } from '@studio/testing/mocks';
```

Reply:
Same here as [my previous comment](https://github.com/Altinn/altinn-studio/pull/12797#discussion_r1611766890)